src/database/queries/Label.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from '@prisma/client';
+import { Label } from '../types';
+
+/**
+ * Retrieves all available labels from the datastore.
+ *
+ * @param db
+ */
+export async function getLabels(db: PrismaClient): Promise<Label[]> {
+  return db.label.findMany({
+    orderBy: { name: 'asc' },
+    select: {
```

Comment:
I thought it would be nice to return the labels in alphabetical order for (some sort of) consistency), even if we only have one label in our sights for the foreseeable future.